bool getpoint
---
int32 num
geometry_msgs/Point point1
geometry_msgs/Point point2
geometry_msgs/Point point3
geometry_msgs/Point point4
geometry_msgs/Point point5
geometry_msgs/Point point6